src/js/consts/cycleCountReportingIndicators.js

```diff
@@ -0,0 +1,41 @@
+const cycleCountReportingIndicators = {
+  totalCount: {
+    title: 'react.cycleCount.indicators.cardTitle.totalCount.label',
+    titleDefaultValue: 'Total count physical inventory',
+    numberType: 'number',
+    subtitle: 'react.cycleCount.indicators.cardSubtitle.inMyInventory.label',
+    subtitleDefaultValue: 'in my inventory',
+    info: 'react.cycleCount.indicators.cardInfo.totalCount.label',
+    infoDefaultValue: 'Total physical inventory count',
```

Comment:
title -> titleLabel
titleDefaultValue -> defaultTitle
numberType -> not sure what it means
subtitle -> subtitleLabel
subtitleDefaultValue -> defaultSubtitle
info -> infoLabel
infoDefaultValue -> defaultInfo